tssc/config/decryptors/sops.py

```diff
@@ -0,0 +1,152 @@
+"""ConfigValueDecryptor that uses SOPS to decyrpt ConfigValues
+
+Also See
+--------
+https://github.com/mozilla/sops
+"""
+
+from io import StringIO
```

Comment:
I wonder if there would be a performance difference using cstringio

Reply:
future us problem?

Reply:
Agreed. It was more of a case of dumping something I've seen before be a performance issue as part of a master's project. I highly doubt it would impact this, but best to plant the seed of the idea in your mind for the future. :)